### PR TITLE
add dummybuild to gitignore

### PR DESCRIPTION
**Pull request by mozrokafor**
## Description

    Add dummybuild to git ignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -47,6 +47,7 @@ nebula/libnebula.a
 src/mozillavpn_autogen/
 /build-*/
 /build/
+dummybuild
```
Comment (bakulf) [inline .gitignore:50]: where does this dummybuild come from?

Comment (mozrokafor) [reply .gitignore:50]: cmake command ran before being able to run a test

cmake -S . -B $(pwd)/dummybuild -DBUILD_DUMMY=ON 
cmake --build $(pwd)/dummybuild -j$(nproc)

Comment (mozrokafor) [reply .gitignore:50]: functional test*

Comment (bakulf) [reply .gitignore:50]: Right but you can call this dir `build` too. Why dummybuild?

Comment (bakulf) [reply .gitignore:50]: Is it written in the README.md?

**Pull request merged**